notebooks/1.1_Data_Ingestion-Intake.ipynb

```diff
@@ -27,9 +27,7 @@
    "source": [
```

Comment:
<p>I'm getting a bizarre error:</p><p> </p><p><code></code><code></p><p>ValueError: red not found in list of eo:bands in collection.Valid values: ['OLI_B1', 'OLI_B2', 'OLI_B3', 'OLI_B4', 'OLI_B5', 'OLI_B6', 'OLI_B7', 'TIRS_B10', 'blue', 'coastal', 'green', 'lwir11', 'nir08', 'red', 'swir16', 'swir22']</p><p></code><code></code></p><p> </p><p>obviously 'red' is in this list.. so I'm confused. Same error for 'green' if I remove 'red' from the requested bands.. so something wrong with parsing the collection values</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ProjectPythia/landsat-ml-cookbook/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>